hvplot/converter.py

```diff
@@ -466,9 +472,12 @@ def __init__(
                     ylim = (y0, y1)
 
         # Operations
+        self.aggregation_threshold = aggregation_threshold
         self.datashade = datashade
         self.rasterize = rasterize
         self.downsample = downsample
+        if self.aggregation_threshold and not (self.datashade or self.rasterize):
```

Comment:
TBH I prefer explicit over implicit and have users specify what operation they want, do you have a strong argument for the operation be set automatically?

Reply:
If we keep the implemented behavior, the docstring of `aggregator_threshold` will have to be updated to reflect that.